docs/development/db-schema.md

```diff
@@ -0,0 +1,60 @@
+# Database Schema
+
+Currently Service Manager supports only PostgreSQL as a database.
+
+The database schema is built using a sequence of SQL scripts located in
+[storage/postgres/migrations](../../storage/postgres/migrations).
+For each schema version there are two scripts - _up_ and _down_.
+So it is possible also to downgrade the schema to a an older version.
+Each file name starts with some numbers - these represent the schema version.
+
+These scripts are executed in a specific or by [migrate](https://github.com/golang-migrate/migrate) library
+during SM start up. This library records in the database the current schema version,
+so subsequent executions do not attempt to recreate the same db objects.
+It also uses db locks to prevent parallel migrations in case multple instances of SM start at the same time.
+
+## Upgrade / downgrade db schema
+Normally SM upgrade the db schea automatically during startup.
+Still in rare cases it is necessary to do this manually. Here is how to do it.
```

Comment:
I think one comma need to be added here:
Still in rare cases -> Still, in rare cases